network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -366,6 +374,49 @@ class ProductPage(AirtableMixin, FoundationMetadataPageMixin, Page):
         related_name='votes',
     )
 
+    @classmethod
+    def map_import_fields(cls):
+        mappings = {
```

Comment:
is there some way to get this from the model itself through some Django magic that taps into verbose name/label/[...]?